src/figury/AnimatorApp.java

```diff
@@ -76,7 +79,43 @@ public void actionPerformed(ActionEvent e) {
 		});
 		btnAnimate.setBounds(100, 239, 80, 23);
 		contentPane.add(btnAnimate);
-		
+
+		// Dodane: suwak zmieniający wartość prędkości animacji dla nowo dodanego kształtu
+		// (skala suwaka jest lepiej widoczna przy powiększeniu okienka)
+		JSlider slider = new JSlider();
+		slider.setPaintTicks(true);
+		slider.setPaintLabels(true);
+		slider.setMajorTickSpacing(5);
+		slider.setOrientation(SwingConstants.HORIZONTAL);
+		slider.setMinimum(1);
+		slider.setMaximum(151);
+		slider.addChangeListener(new ChangeListener() {
+			public void stateChanged(ChangeEvent e) {
+				kanwa.changeDelay(slider.getValue());
```

Comment:
to zmienia tylko szybkość wyświetlania. Szybkość animacji trzeba gdzie indziej zmienić